2/contrib/s2i/run

```diff
@@ -60,6 +60,18 @@ function create_jenkins_config_from_templates() {
 }
 
 function install_plugins() {
+  mkdir -p ${JENKINS_HOME}/plugins
+    
+  if [ "$(ls -A /usr/lib/jenkins/*.hpi 2>/dev/null)" ]; then
+      # found rhel rpm installed plugins in the 3.7 or later location
+      echo "Copying $(ls /usr/lib/jenkins/*.hpi | wc -l) RHEL RPM installed Jenkins plugins to ${JENKINS_HOME} ..."
```

Comment:
remove the references to RHEL since this script is shared w/ the centos image.

Reply:
this path though will only execute on rhel ... there will be no `/usr/lib/jenkins/*.hpi` files in the centos image

Reply:
unless someone has extended the image and installed their own plugins from rpms on a centos image, but is still using our run script....

Reply:
ok i'm converted removing "RHEL"